integration-cli/docker_cli_push_test.go

```diff
@@ -84,31 +84,6 @@ func testPushMultipleTags(c *check.C) {
 	dockerCmd(c, "tag", "busybox", repoTag2)
 
 	dockerCmd(c, "push", repoName)
-
-	// Ensure layer list is equivalent for repoTag1 and repoTag2
-	out1, _ := dockerCmd(c, "pull", repoTag1)
```

Comment:
I think this should be `push` instead of `pull`; this message is printed when pushing an image; https://github.com/moby/moby/blob/8e610b2b55bfd1bfa9436ab110d311f5e8a74dcb/registry/session.go#L516